Marlin/src/lcd/ultralcd.h

```diff
@@ -259,15 +259,11 @@ class MarlinUI {
   }
 
   #if HAS_BUZZER
-    static inline void buzz(const long duration, const uint16_t freq) {
-      #if ENABLED(LCD_USE_I2C_BUZZER)
-        lcd.buzz(duration, freq);
-      #elif PIN_EXISTS(BEEPER)
-        buzzer.tone(duration, freq);
-      #elif ENABLED(PCA9632_BUZZER)
-        pca9632_buzz(duration, freq);
-      #endif
-    }
+    static void buzz(const long duration, const uint16_t freq);
```

Comment:
`lcd.buzz` and `pca9632_buzz` are lost.

Reply:
At the moment it is also very hard to understand code, especially with `ultralcd_ *`